lib/flapjack/notifier.rb

```diff
@@ -120,7 +121,7 @@ def process_notification(notification)
             memo[alert] = {
               'duration' => last_change ? (Time.now.to_i - last_change) : nil,
               'state'    => ec.state,
-              'summary'  => ec.summary
+              'summary' => ec.summary
```

Comment:
It had been removed in the original PR, so I'd restored by re-typing. :) Will clear up once merged.